test/test_data/BUILD

```diff
@@ -59,3 +60,8 @@ wasm_rust_binary(
     srcs = ["random.rs"],
     wasi = True,
 )
+
+proxy_wasm_cc_binary(
+    name = "configure_check.wasm",
+    srcs = ["configure_check.cc"],
```

Comment:
Nit: consider renamig to `canary_check`.

Reply:
Done